### DISCUSSION TITLE
Welcome to rwallet Discussions!

### DISCUSSION THREAD

**Discussion by patogallaiovlabs** [rsksmart/rwallet#681] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bitcoinuser** [comment]
Hi, I'm Leo. I really like Rwallet. I heard that the team that was working on the project was out and I didn't like it. Hope the project continue to be developed like before. I translated Rwallet to PT-BR and really gave many suggestions and reported many bugs.
